### PR TITLE
test: Fixup the user home directory also when running tests (in Docker)

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

**Review by heliocastro — APPROVED** [review]
Sounds's right

File: utils/test/src/main/kotlin/ProjectConfig.kt [hunk 1]
```diff
@@ -23,12 +23,14 @@ import io.kotest.core.config.AbstractProjectConfig
...
 import io.kotest.extensions.junitxml.JunitXmlReporter
 
+import org.ossreviewtoolkit.utils.common.Os
 import org.ossreviewtoolkit.utils.ort.OrtProxySelector
 
 class ProjectConfig : AbstractProjectConfig() {
     override val specExecutionOrder = SpecExecutionOrder.Annotated
 
     init {
+        Os.fixupUserHomeProperty()
```
Comment (fviernau) [inline utils/test/src/main/kotlin/ProjectConfig.kt:33]: Why is this necessary? (Does it fix any issue?)

**Review by fviernau — APPROVED** [review]

File: utils/test/src/main/kotlin/ProjectConfig.kt (hunk 1 above)
Comment (sschuberth) [reply utils/test/src/main/kotlin/ProjectConfig.kt:33]: It does not fix any issue in our current setup, but when using a different JDK (and maybe some other base image) then running tests in Docker could fail as the `user.home` Java system property bogusly is set to `?`. As this is rather hard to track down, I'm proactively addressing it here.
